SWLOR.Game.Server/Feature/PerkDefinition/MartialArtsPerkDefinition.cs

```diff
@@ -435,5 +452,54 @@ private void LegSweep()
                 .GrantsFeat(FeatType.LegSweep3);
         }
 
+        private void FlurryStyle()
+        {
+            _builder.Create(PerkCategoryType.MartialArtsStaff, PerkType.FlurryStyle)
+                .Name("Flurry Style")
+                .TriggerEquippedItem((player, item, slot, type, level) =>
+                {
+                    if (slot != InventorySlot.RightHand) return;
+
+                    Stat.ApplyAttacksPerRound(player, item);
+                })
+                .TriggerUnequippedItem((player, item, slot, type, level) =>
+                {
+                    if (slot != InventorySlot.RightHand) return;
+
+                    Stat.ApplyAttacksPerRound(player, OBJECT_INVALID);
+                })
+
+                .AddPerkLevel()
+                .Description("Favoring alacrity over brawn, attacks with staves now use your Agility for accuracy and Perception for damage. In addition, you gain an additional attack with staves, but all staff attacks are made with a -10% to-hit penalty.")
+                .Price(1)
+                .RequirementCannotHavePerk(PerkType.CrushingStyle)
+                .GrantsFeat(FeatType.FlurryStyle)
+
+                .AddPerkLevel()
+                .Description("You master the art of striking quickly and often. You gain an additional attack with staves, and no longer suffer a to-hit penalty for attacks made with staves.")
+                .Price(4)
+                .RequirementSkill(SkillType.MartialArts, 35)
+                .RequirementCharacterType(CharacterType.Standard)
+                .GrantsFeat(FeatType.FlurryMastery);
+        }
+        private void CrushingStyle()
+        {
+            _builder.Create(PerkCategoryType.MartialArtsStaff, PerkType.CrushingStyle)
+                .Name("Crushing Style")
+
+                .AddPerkLevel()
+                .Description("Attacking with overwhelming strength, the MGT bonus to damage for attacks with MGT weapons is increased to your modifier for attacks with a staff. In addition, your critical chance is raised by 15%.")
```

Comment:
I find this wording a little confusing. Maybe something along the lines of "The MGT weapon bonus is increased by your modifier..." is more clear?